packages/components/src/components/types/multi-step.types.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export type TMultiStepRefProps = {
+    // TODO: Remove the type defined in MT5 and reference this one instead
+    goNextStep: () => void;
+    goPrevStep: () => void;
+};
+
+export type TMultiStepProps = {
+    className?: string;
+    lbl_previous?: string;
+    steps: Array<{ component: React.ReactNode }>;
```

Comment:
just a small question, if it is component, maybe we better should use here React.reactElement? because ReactNode includes also strings, number, null, undefined etc...

Reply:
Yeah @yauheni-kryzhyk-deriv, I had put `ReactNode` because of stories, but I guess we can update that to `divs` instead of strings